app/controllers/application_controller/ci_processing.rb

```diff
@@ -235,6 +235,14 @@ def check_scan_requirements(selected_items)
     true
   end
 
+  def check_reset_requirements(selected_items)
+    if VmOrTemplate.find(selected_items).any? { |vm| !vm.supports_reset? }
+      javascript_flash(:text => _("Reset does not apply to selected item"), :severity => :error, :scroll_top => true)
```

Comment:
Changed to "Reset does not apply to at least one of the selected items" which seems a common phrasing in the project